edx_notifications/server/web/static/edx_notifications/js/models/counter_icon_model.js

```diff
@@ -0,0 +1,11 @@
+var CounterIconModel = Backbone.Model.extend({
```

Comment:
It would be best if your JS code could use RequireJS. Look at how content search used it, for example:

https://github.com/edx/edx-platform/blob/master/lms/static/js/search/models/search_result.js

If not, then our best practice is to put everything into the edX namespace. Here's a simple example:

https://github.com/edx/edx-platform/blob/master/lms/static/js/student_account/models/LoginModel.js